cli/src/main/resources/help/module.txt

```diff
@@ -3,7 +3,7 @@ SYNOPSIS
   To add a module locally
 
     module add [--module-root-dir=modules_root_directory_path] --name=module_name 
-               [--slot=slot_name] --resources=list_of_jars [--resource-delimiter=delimiter]
+               [--slot=slot_name] --(absolute-)resources=list_of_jars [--resource-delimiter=delimiter]
```

Comment:
Could you please list the new argument separately next to the resources argument?
It's quite tricky to express the dependencies between the arguments here in this syntax. I'm wondering whether it'd be less confusing to mark both of them as optional.
Thanks for the contribution!